src/Standards/Squiz/Tests/Commenting/VariableCommentUnitTest.inc

```diff
@@ -402,3 +402,19 @@ class ReadOnlyProps
 
      private readonly string $variable;
 }
+
+class UnionTypes
+{
+    /**
+     * @var array|boolean
+     */
+    private array|bool $variableName = array();
+}
+
+class IntersectionTypes
+{
+    /**
+     * @var \Iterator|\Countable
+     */
+    private \Iterator&\Countable $variableName;
```

Comment:
The comment says `|` but the code says `&`. Is this intentional?

Reply:
There is a difference between type hinting and doc types. As far as I know there is (currently) no such thing like intersection types, when it comes to doc-types

Reply:
I'm fine with the doctype as it is. After all, this test is _not_ about the type used in the docblock, but about the docblock not being found when PHP native union/intersection types are being used.